flatgfa/src/flatbed.rs

```diff
@@ -0,0 +1,191 @@
+use zerocopy::{AsBytes, FromBytes, FromZeroes};
+use crate::memfile::MemchrSplit;
+use std::io::BufRead;
+use atoi::FromRadix10;
+use bstr::BStr;
+use crate::pool::{Pool, Span, Store, HeapStore, FixedStore, Id};
+
+#[derive(Debug, FromZeroes, FromBytes, AsBytes, Clone, Copy)]
+#[repr(packed)]
+pub struct BEDEntry {
+    pub name: Span<u8>,
+    pub start: u64,
+    pub end: u64,
+}
+
+pub struct FlatBED<'a> {
```

Comment:
A comment that could work here: "A flat representation of an entire BED file, i.e., a list of named intervals."